manifests/crane/params.pp

```diff
@@ -0,0 +1,10 @@
+# Default params for Crane
+class pulp::crane::params {
+  $ca_cert  = undef
+  $cert = undef
+  $data_dir = '/var/lib/crane/metadata'
+  $data_dir_polling_interval = 60
+  $debug = false
+  $key = undef
```

Comment:
Nitpick - I realize these are in sorted order but the related items might be nice together.